src/python/doc/witness_complex_user.rst

```diff
@@ -69,7 +69,7 @@ The construction of the Euclidean versions of complexes follow the same scheme:
 
 In the non-Euclidean classes, the lists of nearest landmarks are supposed to be given as input.
 
-The constructors take on the steps 1 and 2, while the function 'create_complex' executes the step 3.
+The constructors take on the steps 1 and 2, while the function :func:`!create_complex` executes the step 3.
```

Comment:
It does not make a real link, but I suppose it is because we cannot link to witness_complex_ref, strong_witness_complex_ref, euclidean_witness_complex_ref and euclidean_strong_witness_complex_ref, no ?

Reply:
Yes, at least that was my goal: give it a consistent look, but since I can't make it link to 4 things (and I don't want to add a list of links after the word), don't make it a link at all (with `!`).